anthos-bm-edge-deployment/inventory/host_vars/edge-2.yaml

```diff
@@ -18,4 +18,13 @@ cluster_name: "edge-1"
 machine_label: "edge-2"
```

Comment:
@Shabirmean -- for a public version, we should only provide `cnuc-*.yaml` files (no configuration for physical hosts...they would be a replica of these, but no need to also upkeep that configuration)

Reply:
True. Deleted all the non `cnuc` files